### ISSUE TITLE
Feedback Group should be linked with the field has the error

### ISSUE THREAD

**Issue by marcoscv-work** [liferay/clay#5247]
https://storybook.clayui.com/?path=/story/design-system-components-input--input-feedback
<img width="392" alt="image" src="https://user-images.githubusercontent.com/7963804/207120492-6a3c8237-b543-4e21-8d0a-91c6eb50a927.png">

When ClayForm.Group has-error. We should add `aria-invalid=""` to ClayInput in order to indicate that input contains an error, and then use aria-describedby to link with the error text (by the unique ID): 
https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA21
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-invalid

If it is something is not controlled by component and cannot be done automatically or on the component context we should provide examples by using the aria roles correctly.

---
**_Not directly related with this suggestion/issue_**

As a good practice: We could add to the form group of the inputs with errors an alert to say, hey, you have errors, you can go to checkthem

In a perfect world:
We could add an alertdialog providing an ordered list of error to navigate them:
https://www.w3.org/WAI/WCAG21/Techniques/general/G139.html#example-2-client-side-error-checking-with-a-popup

Here an old example in a codependent applied to DXP:
https://codepen.io/marcoscv/pen/oRGbXo

**Comment by matuzalemsteles** [comment]
In the second case, I think we would have to implement part of this issue because we don't control the input and validation states yet in Clay, so much of this is still implemented by the developer, the validations. But I think we can explore this when we start working on the issue #4741.
